### ISSUE TITLE
Gallery view of search results

### ISSUE THREAD

**Issue by caseyedavis12** [WGBH/AAPB2#1131]
In a previous Blacklight update, we lost the Gallery view feature on the search results page. 

**waiting on:**
  - [ ] mocks

**Done when**
- [ ] We have the option to toggle between list view and gallery view on the search results page.  
- [ ] Gallery view displays images and title only.

**Comment by sroosa** [comment]
Duplicate of #1424
